src/lib/lib_cron.py

```diff
@@ -63,11 +63,18 @@ def install_cronjob(self):
 
             logrotate_unit = hookenv.local_unit()
             python_venv_path = os.getcwd().replace("charm", "") + ".venv/bin/python3"
+            if hookenv.has_juju_version("3.0"):
```

Comment:
(doc, non-blocking): it's good to briefly explain that why `3.0` here and what this `if-else` does, because without looking at the source code, I thought it was only looking for `juju == 3.0`